packages/app/src/pages/Specs/Runner.vue

```diff
@@ -147,6 +147,24 @@ iframe.aut-iframe {
   background: white;
 }
 
+.is-screenshotting #main-pane {
+  overflow: auto !important;
+}
+
+.is-screenshotting.screenshot-scrolling #main-pane {
+  overflow: visible !important;
+}
+
+#resizable-panels-root {
+  overflow-x: auto;
+  overflow-y: hidden;
+}
+
+.is-screenshotting #resizable-panels-root {
+  overflow-x: visible;
+  overflow-y: visible;
+}
```

Comment:
I'd actually like to consolidate more of our screenshot css rules into this structure, instead of conditionally managing tailwind classes with JS. But to keep this PR focused, I'm not doing that here, just moving what's needed.